### ISSUE TITLE
Ideas for the Structure or Outline of the Platform

### ISSUE THREAD

**Issue by arijeetsat** [p-society/IIIT-forum#1]
Lets give our ideas and suggestion about this project.

**Comment by arijeetsat** [comment]
This can be one example : http://forum.iiit.ac.in/2017/?page_id=14

**Comment by geekysrm** [comment]
Yeah, guys
Please give ideas as to how it would look and what it should contain

**Comment by sudoankit** [comment]
If hosting isn't an issue, go for [Discourse](https://www.discourse.org). It's simple, minimal, has cross platform apps and [easy to setup](https://github.com/discourse/discourse/blob/master/docs/INSTALL.md). 

IHMO, integrate this to programming society so you need one domain and can use the subdomain for Discourse/IIIT Forum. Just get good web hosting space, AWS or Linode, etc.

**Comment by geekysrm** [comment]
Yeah, I had seen discourse earlier. It is free and open source for self-hosting. I had seen it freecodecamp, Hasura, Udacity using it.

We just need to buy good hosting.
